administrators.rst

```diff
@@ -671,7 +671,8 @@ S3 (Amazon)
 +++++++++++
 
 Amazon S3 is available as an access protocol as of Storage Service 0.12. At
-this time, locations within S3 have only been tested for AIP Storage.
+this time, locations within S3 have only been tested for AIP Storage and as a
+Transfer Source.
```

Comment:
I think I'm tempted to remove the allusion to testing and say that S3 can be used for AIP storage, replication, and as a transfer source. IDK what do you think? I think we're getting more and more confident of the integration?

The replication reference comes from a recent PR https://github.com/artefactual/archivematica-storage-service/pull/495 so we can do that now too! :)

Reply:
@ross-spencer I agree in principle - I think we've tested sufficiently. The language comes from our writing, rather than @helenst's, so I'm not fussed if it gets changed here or via separate PR!